org.virt_manger.virt-viewer.json

```diff
@@ -0,0 +1,287 @@
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "libgcrypt-config",
+                    "commands": [
+                        "if [ $1 == '--version' ]; then set -- '--modversion' ${@:4}; fi",
+                        "exec pkg-config $@ libgcrypt"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "gtk-vnc",
+            "cleanup": [
+                "/bin"
+            ],
+            "sources":[
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtk-vnc/0.7/gtk-vnc-0.7.2.tar.xz",
+                    "sha256": "f893f32b1ef6d09dd23cda39b8a2567be7c2aebda58026288e1362f042e20808"
+                }
+            ]
+        },
+        {
+            "name": "spice-protocol",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/spice/spice-protocol.git",
+                    "tag": "v0.12.13",
+                    "commit": "8dda82b49d8f848a25e3a1ef6df943276c59e462"
+                }
+            ]
+        },
+        {
+            "name": "libusb",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libusb/libusb/archive/v1.0.22.tar.gz",
+                    "sha256": "3500f7b182750cd9ccf9be8b1df998f83df56a39ab264976bdb3307773e16f48"
+                }
+            ]
+        },
+        {
+            "name": "usbutils",
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/gregkh/usbutils.git",
+                    "tag": "v009",
+                    "commit": "133f781d88ae4aee22654bfed25894518e4109c7"
+                }
+            ]
+        },
+        {
+            "name": "usbredir",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://anongit.freedesktop.org/spice/usbredir",
+                    "tag": "usbredir-0.7.1",
+                    "commit": "e1a7e3dbbe091bfdc568372ff5ab18ed7eae972e"
+                }
+            ]
+        },
+        {
+            "name": "libcacard",
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://anongit.freedesktop.org/spice/libcacard",
+                    "tag": "v2.5.3",
+                    "commit": "aaa5251791bf0b1640afcba77a7d79ea23c42d53"
+                }
+            ]
+        },
+        {
+            "name": "lz4",
+            "build-options": {
+                "env": {
+                    "PREFIX": "/app"
```

Comment:
You use ${FLATPAK_DEST} above.

Reply:
So are you saying to replace `PREFIX` with `FLATPAK_DEST`?

Reply:
No, see module `gcrypt-wrapper`. I meant: Use `/app` or `${FLATPAK_DEST}` consistently.

Reply:
does https://github.com/flathub/flathub/pull/391/commits/72a693627838a6fd053fc4dbb4274fe8f8afe268 solve this?

Reply:
Sure. Not that it does have any effect, it was only a minor style thing.

Reply:
Thanks, you've been doing a great job of helping me better my manifest.